### ISSUE TITLE
Empty chapter shown if all questions have visibleExpression false

### ISSUE THREAD

**Issue by fdlk** [molgenis/molgenis#8561]
### How to Reproduce
Make a questionnaire with two chapters (compound attributes)
Each chapter has one question.
The second chapter and question have visibleExpression set to the answer to the first question.
Start the questionnaire
Answer the first question with false

### Expected behavior
The second chapter is not shown

### Observed behavior
The second chapter is shown and empty.
It is not immediately clear how to finish the questionnaire

**Comment by connoratrug** [comment]
Not sure about the ‘Expected behavior’ here, i would expect;
- chapter 2 to be empty
- form completion at 100%
- submit button enabled 
- a message in the empty chapter indicating all items are done / there a no items 

this way magically disappearing and reappearing chapter are prevented.
See this more as the start of a feature request, but same difference imo

**Comment by fdlk** [comment]
The chapter itself also has visibleExpression set to false, so I'd expect it not to be visible.

**Comment by connoratrug** [comment]
@fdlk  yes then i agree, could you add a minimal emx showing the issue ?

**Comment by fdlk** [comment]
I believe @joerivandervelde has one

**Comment by connoratrug** [comment]
@fdlk checked with @joerivandervelde , the emx can not be made public. Still think we need a public emx / example that show what the problem is and can be used to verify the fix/feature.
